apps/test/unit/templates/progress/BubbleFactoryTest.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import {shallow, mount} from 'enzyme';
+import {updateQueryParam} from '../../../../src/code-studio/utils';
+import {currentLocation} from '@cdo/apps/utils';
```

Comment:
Q: I've often been curious about these two different import path formats. Would `@cdo/apps/utils/code-studio/utils` have worked for `updateQueryParam`? I've tended to use the shortcut path when I see lots of repeating `../` in the relative path, but I'm not sure if there are other schools of thought.

Reply:
Good question! I know this has come up in the past (e.g. https://codedotorg.slack.com/archives/C0T0PNR0D/p1563236491128200) and as far as I know we haven't pushed for or landed on standardization. I was trained under the old regime though so if someone's interested in championing consistency here I can get on board!